CHANGELOG.md

```diff
@@ -16,7 +16,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 ### Fixed
+
 ### Removed
+- Removed `lutimes` on OpenBSD as it is not implemented on OpenBSD.
```

Comment:
The Removed section is for functionality that's been removed.  These functions never worked in the first place, so I think they should probably go under "Fixed".

Reply:
I've put the changelog entries under "Fixed".